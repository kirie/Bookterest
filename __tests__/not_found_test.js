import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from '../src/components/not_found';

jest.mock('react-dom');

describe('Not Found Component (Snapshot)', () => {
  it('Not Found Renders', () => {
    const component = renderer.create(<NotFound />);
    const json = component.toJSON();
    expect(json).toMatchSnapshot();
  });
});
