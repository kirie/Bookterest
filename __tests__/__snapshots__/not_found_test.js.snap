exports[`Not Found Component (Snapshot) Not Found Renders 1`] = `
<div>
  <h1>
    Oops!  That page doesn\'t exist.
  </h1>
</div>
`;
